README.md

```diff
@@ -2071,10 +2070,12 @@ final class ValidTest extends TestCase
 | :information_source: **INFORMATION:** |
 |:--------------------------------------|
 
-- It's better to avoid a shared state between tests.
-- To reuse elements between a few tests:
-    * private factory methods - reusing in one class (like above)
-    * [Object mother](#object-mother) or [Builder](#builder) - reusing in a few classes
+The usage of private methods improves readability and clarifies separation. 
+In the same time, additional effort is needed to use them inside particular testing method.  
+
+To share similar testing objects use:
```

Comment:
How about some clarification?  
**To share similar testing objects between multiple test classes use:**